simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListRegistryViewModel.kt

```diff
@@ -189,7 +194,29 @@ class AccountListRegistryViewModel(private val locationManager: LocationManager)
         .filter { it.id != accountRegistry.defaultProvider.id }
         .toMutableList()
       updatedProvidersList.removeAll(usedAccountProviders)
-      accountProvidersList.onNext(updatedProvidersList)
+
+      val featuredLibrariesList = updatedProvidersList.filter {
+        featuredLibrariesList.contains(it.id.toString())
```

Comment:
Shadowing `featuredLibrariesList` like this is confusing. I would at least put `this.` on the inner usage to make it clearer, but the best option is to use a different name for the local variable. (This may become moot if you move the definition of the featured libraries out of this file.)

Reply:
You're right. I didn't even notice the names were the same and that's why I kept it like this. I'll fix this